crates/prql-compiler/src/semantic/resolver/transforms.rs

```diff
@@ -1071,4 +1078,18 @@ mod tests {
             - Wildcard
         "###);
     }
+
+    #[test]
+    fn test_prql_version() {
+        println!(
```

Comment:
Can we just run `.unwrap` to ensure it passes, rather than `println!`?

(or possibly we don't need this test given we have https://github.com/PRQL/prql/pull/3533/files#diff-35eb52ba10a796006603d51870d6532a65ad5d351ed21580bab2e0548c360a34R3406 — your call on adjusting vs removing